.nvmrc

```diff
@@ -0,0 +1 @@
+12.18.1
```

Comment:
why do we need this ?

Reply:
The screwdriver.yaml was changed to 12. When I initially tried to run the tests they failed because I was running node 10. 
I always look for a .nvmrc file because I use nvm all over the place in various places. 
`nvm use` will now switch my node version to 12 when i work on this code.
It is not a requirement, and I can remove it, but the tests fail when running on node 10 at this point.